xbmc/PlayListPlayer.cpp

```diff
@@ -983,8 +984,20 @@ void PLAYLIST::CPlayListPlayer::OnApplicationMessage(KODI::MESSAGING::ThreadMess
           {
             return;
           }
-          if (MUSIC::IsAudio(*item) || IsVideo(*item))
+          const bool isVideo{IsVideo(*item)};
```

Comment:
i guess it is `VIDEO::IsVideo` being called here

```suggestion
          const bool isVideo{VIDEO::IsVideo(*item)};
```

Reply:
Done